packages/apps/plugins/plugin-script/src/ScriptPlugin.tsx

```diff
@@ -100,30 +101,30 @@ export const ScriptPlugin = ({ containerUrl }: ScriptPluginProps): PluginDefinit
             case 'main':
               return isObject(data.active, ScriptType.schema, ScriptType.filter()) ? (
                 <Main.Content classNames={[baseSurface, fixedInsetFlexLayout, topbarBlockPaddingStart]}>
-                  <ScriptBlock
-                    id={(data.active as any).id}
-                    source={(data.active as ScriptType).source}
-                    classes={{ toolbar: 'px-2' }}
+                  <ScriptBlockWrapper
+                    //
```

Comment:
//

Reply:
this is stop prettier from making the code less pretty.